File: snippets/java/M07_PriceTax.java
import inout.Out;

class PriceTax {
    public static void main(String[] args) {
        double[] prices = {12.3, 5.2, 8.7, 1.2, 8.0};
        double[] gross = new double[5];

        for (int i = 0; i < prices.length; i++) {
            gross[i] = prices[i] * 1.2;
        }

        for (double price : gross) {
            Out.println(price);
        }
    }
}

File: snippets/python/m07_price_tax.py
prices = [12.3, 5.2, 8.7, 1.2, 8.0]
gross = []

for price in prices:
    gross.append(price * 1.2)

print(gross)

# or, with list comprehension
prices = [12.3, 5.2, 8.7, 1.2, 8.0]
gross = [price * 1.2 for price in prices]

print(gross)
